src/serviceClients/sqliteClient.ts

```diff
@@ -28,10 +29,23 @@ export class SQLiteClient {
     this.fullPath = join(this.layerSourcesPath, originDirectory, this.packageName);
   }
 
+  public getDB(fileMustExistFlag: boolean): SQLiteDB {
+    try {
+      return new Database(this.fullPath, { fileMustExist: fileMustExistFlag });
+    } catch (err) {
+      const message = `cant create SQLiteDB`;
```

Comment:
`Can't open file {this.fullPath} as SQLiteDB`

Reply:
done